[docs] Matt review (#234)
mui-org/material-ui-x: docs/src/pages/components/data-grid/rows/rows.md

## Row height

By default, the rows have a height of 52 pixels.

> It matches the normal Material Design height of the [specification](https://material.io/components/data-tables).

Reviewer: ```suggestion
This matches the normal height in the [Material Design guidelines](https://material.io/components/data-tables).
```
